ch3/01.py

```diff
@@ -0,0 +1,8 @@
+import random
+
+lista = []
+
+for x in range(3):
```

Comment:
x is not used here, why create it? can you do it without creating a new variable?